src/tools/erthost/erthost.cpp

```diff
@@ -154,6 +238,10 @@ int main(int argc, char* argv[], char* envp[])
     const char* const env_simulation = getenv("OE_SIMULATION");
     const bool simulation = env_simulation && *env_simulation == '1';
 
+    const char* const env_log_detailed = getenv("OE_LOG_DETAILED");
```

Comment:
Maybe we should auto enable this for the VERBOSE log level? I feel it would be pretty redundant to have two environment variables to enable full logging, especially since I think having a call trace is properly desired at VERBOSE logging level.

Also, documentation please :P